libs/libcommon/src/libcommon/config.py

```diff
@@ -120,6 +120,7 @@ class ProcessingGraphConfig:
             "/parquet": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
             "/dataset-info": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
             "/sizes": {"input_type": "dataset", "requires": "/parquet-and-dataset-info"},
+            "/basic-stats": {"input_type": "split", "requires": "/parquet-and-dataset-info"},
```

Comment:
you could directly depend on /parquet since you don't use the dataset-info part in the compute function